<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabBatchResize</class>
 <widget class="QWidget" name="TabBatchResize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批量缩放</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutInput">
     <item>
      <widget class="QLabel" name="labelInputFolder">
       <property name="text">
        <string>输入文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineInputFolder"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectInputFolder">
       <property name="text">
        <string>浏览...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutOutput">
     <item>
      <widget class="QLabel" name="labelOutputFolder">
       <property name="text">
        <string>输出文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineOutputFolder"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectOutputFolder">
       <property name="text">
        <string>浏览...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listImages">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>214</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutListOp">
     <item>
      <widget class="QPushButton" name="btnReloadImage">
       <property name="text">
        <string>重新加载</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddImages">
       <property name="text">
        <string>添加图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveImages">
       <property name="text">
        <string>移除选中图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearList">
       <property name="text">
        <string>清空列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSavePath">
       <property name="text">
        <string>保存路径</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupResizeMode">
     <property name="title">
      <string>缩放方式</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutMode">
      <item>
       <widget class="QRadioButton" name="radioScale">
        <property name="text">
         <string>按比例缩放</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutScale">
        <item>
         <widget class="QLabel" name="labelScale">
          <property name="text">
           <string>比例(%)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinScale">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radioFixedWidth">
        <property name="text">
         <string>固定宽度等比缩放</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutFixedWidth">
        <item>
         <widget class="QLabel" name="labelWidth">
          <property name="text">
           <string>宽度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinWidth">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>800</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radioFixedHeight">
        <property name="text">
         <string>固定高度等比缩放</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutFixedHeight">
        <item>
         <widget class="QLabel" name="labelHeight">
          <property name="text">
           <string>高度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinHeight">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>600</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnBatchResize">
     <property name="text">
      <string>批量缩放</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string>状态：等待操作</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
